File: src/main/java/at/hannibal2/skyhanni/utils/jsonobjects/ParkourJson.java
package at.hannibal2.skyhanni.utils.jsonobjects;

import at.hannibal2.skyhanni.utils.LorenzVec;

import java.util.ArrayList;
import java.util.List;

public class ParkourJson {
    public List<LorenzVec> locations;
    public List<ShortCut> shortCuts = new ArrayList<>();

    public static class ShortCut {
        public int from;
        public int to;
    }
}

File: src/main/java/at/hannibal2/skyhanni/data/jsonobjects/repo/ParkourJson.kt
package at.hannibal2.skyhanni.data.jsonobjects.repo

import at.hannibal2.skyhanni.utils.LorenzVec
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ParkourJson(
    @Expose val locations: List<LorenzVec>,
    @Expose @SerializedName(value = "short_cuts", alternate = ["shortCuts"]) val shortCuts: List<ParkourShortCut> = listOf(),
)

data class ParkourShortCut(
    @Expose val from: Int,
    @Expose val to: Int,
)
